onnxruntime/core/optimizer/layer_norm_fusion.cc

```diff
@@ -66,6 +76,22 @@ Status LayerNormFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level,
         !IsSupportedDataType(reduce_mean_node)) {
       continue;
     }
+
+    // verify axes is contiguous if more than one elements
```

Comment:
This looks unrelated to the cast issue tho. Please add test cases for valid and invalid `axes` attributes.

Reply:
adding one UT for invalid case, valid case has been covered.